src/utils.js

```diff
@@ -59,7 +60,31 @@ function getFetchOptions(options) {
   return fetchopts;
 }
 
+/**
+ * Create an error response.
+ */
+function createErrorResponse({
+  e, msg, status, log,
+}) {
+  const message = (e && e.message) || msg;
+  if (log) {
+    const args = [message];
+    if (e) {
+      args.push(e, e.stack);
+    }
+    log.error(...args);
+  }
+  /* istanbul ignore next (default 500 status) */
+  return new Response('', {
+    status: (e && e.status) || status || 500,
+    headers: {
+      'x-error': message,
```

Comment:
We have a clean up headers method somewhere in helix-shared that can prevent errors due to unicode characters in error messages. It would be best to use this.

Reply:
Great tip, thanks a lot, done!